index.js

```diff
@@ -10,7 +10,7 @@ module.exports.subscribe = (event, callback) => {
   const build = module.exports.eventToBuild(event.data.data);
 
   // Skip if the current status is not in the status list.
-  const status = module.exports.status || ['SUCCESS', 'FAILURE', 'INTERNAL_ERROR', 'TIMEOUT'];
+  const status = module.exports.status || ['WORKING', 'SUCCESS', 'FAILURE', 'INTERNAL_ERROR', 'TIMEOUT'];
```

Comment:
I'd rather leave the user the choice by setting `export GC_SLACK_STATUS="SUCCESS FAILURE TIMEOUT INTERNAL_ERROR"` (see readme)

Reply:
Sure, good idea!